src/components/sidenav/sidenav.js

```diff
@@ -240,7 +241,11 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
         $mdMedia: $mdMedia
       });
     };
-    var backdrop = !angular.isDefined(attr.mdDisableBackdrop) ? $mdUtil.createBackdrop(scope, "_md-sidenav-backdrop md-opaque ng-enter") : undefined;
+
+    // Only create the backdrop if the backdrop isn't disabled.
+    if (!$mdUtil.parseAttributeBoolean(attr.mdDisableBackdrop)) {
```

Comment:
@ThomasBurleson You are right, but I thought parsing for a boolean value is more consistent and provides a little advantage. This allows the use a interpolated boolean as a value for the attribute.

But I agree with you, negated values doesn't make any sense.